src/material/form-field/directives/floating-label.ts

```diff
@@ -43,6 +92,17 @@ export class MatFormFieldFloatingLabel {
   get element(): HTMLElement {
     return this._elementRef.nativeElement;
   }
+
+  private _startResizeObserver() {
+    if (this._platform.isBrowser) {
```

Comment:
This might be cleaner if we do the support check when the resize observer is created, rather than on each consumption.

Reply:
Done